app/components/concept-page/question-card.ts

```diff
@@ -15,6 +17,9 @@ interface Signature {
 }
 
 export default class QuestionCardComponent extends Component<Signature> {
+  @service declare confetti: ConfettiService;
+
+  @tracked hasShownConfetti = false;
```

Comment:
```suggestion
  @tracked hasFiredConfetti = false;
```

(Minor change - makes it a bit more obvious what the action was)